tests/integration/gov/keeper/keeper_test.go

```diff
@@ -33,17 +26,19 @@ type KeeperTestSuite struct {
 	legacyMsgSrvr     v1beta1.MsgServer
 }
 
-func (suite *KeeperTestSuite) SetupTest() {
-	app := simapp.Setup(suite.T(), false)
+func initFixture(t *testing.T) *fixture {
```

Comment:
we should leave the comment that explains why we use simapp in an integration test.
Given that now tests are standalone, could you remove the usage of simapp for tests that could work with a depinjected app?

Reply:
Could you re-add the comment :)